pkg/mediationcontainer/client_protobuf_endpoint.go

```
@@ -134,7 +139,7 @@ func (endpoint *ClientProtobufEndpoint) waitForSingleServerMessage() {
 
 	// Parse the input stream using the registered message handler
 	messageHandler := endpoint.GetMessageHandler()
-	messageHandler.parse(rawBytes)
+	parsedMsg, _ := messageHandler.parse(rawBytes)
```

do we want to ignore the error?